### PR TITLE
Documentation on pytest file creation error

### PR DESCRIPTION
**Pull request by martinesparza**
Added documentation on error thrown by pytest in windows systems where the path exceeds 259 characters

### CODE REVIEW

File: src/beneuro_data/spike_sorting.py [hunk 1]
```diff
@@ -25,7 +25,7 @@ def run_kilosort_on_stream(
     sorter_params: Optional[dict] = None,
 ):
     """
-    Run Kilosort 4 on a SpikeGLX recording.
+    Run Kilosort 4 on a SpikeGLX recording. Testing this on linux
```
Comment (bagibence) [inline src/beneuro_data/spike_sorting.py:28]: Do we need this?

File: tests/generate_directory_structure_test_cases.py [hunk 2]
```diff
@@ -42,14 +47,36 @@ def create_directory_structure_from_dict(data: dict, path: str):
...
+                raise WindowsMaxPathError(explanation) from e
+            else:
+                raise e
 
 
 if __name__ == "__main__":
     test_dir_path = os.path.dirname(__file__)
-    yaml_folder = os.path.join(test_dir_path, "number_of_valid_sessions_test_yamls")
+    yaml_folder = os.path.join(test_dir_path,
+                               "number_of_valid_sessions_test_yamls")
```
Comment (bagibence) [inline tests/generate_directory_structure_test_cases.py:79]: Can you please revert this to being on one line so that formatting is consistent?

**Review by bagibence — COMMENTED** [review]
Looks great, thanks! Just tiny comments.

File: src/beneuro_data/spike_sorting.py (hunk 1 above)
Comment (martinesparza) [reply src/beneuro_data/spike_sorting.py:28]: Indeed we dont :)

File: tests/generate_directory_structure_test_cases.py (hunk 2 above)
Comment (martinesparza) [reply tests/generate_directory_structure_test_cases.py:79]: What is the formatting length you have? I have to 79, but I will change it in the project to stay consistent

Comment (bagibence) [reply tests/generate_directory_structure_test_cases.py]: Thanks! It's 92 for black.

**Review by bagibence — APPROVED** [review]
Perfect, thanks!

**Pull request merged**